pkgs/development/compilers/llvm/4/clang/default.nix

```diff
@@ -13,20 +13,32 @@ let
       mv clang-tools-extra-* $sourceRoot/tools/extra
     '';
 
-    buildInputs = [ cmake libedit libxml2 llvm python ];
+    buildInputs = [ cmake libedit libxml2 llvm python python.pkgs.sphinx ];
```

Comment:
Hmm, okay. I'll push that here although it changes the scope of the PR a bit.. feel free to retitle it if you'd like :+1:.